lib/modules/apostrophe-pages/lib/api.js

```diff
@@ -651,9 +664,22 @@ module.exports = function(self, options) {
     // also change the order of subpages if you can
     // edit the subpage you're moving. Otherwise you
     // must have edit permissions for the new parent page.
-    if ((data.oldParent._id !== data.parent._id) && (data.parent.type !== 'trash') && (!data.parent._edit)) {
+    if (
+      (data.oldParent._id !== data.parent._id) &&
+      (data.parent.type !== 'trash') &&
+      (!data.parent._edit)
+    ) {
       return callback('forbidden');
     }
+
+    if (
+      workflow &&
+      workflow.options.autoCommitPageMoves &&
+      !self.apos.permissions.can(req, 'admin-apostrophe-page', moved)
```

Comment:
Are you sure this verifies the user can commit in the locale?

Reply:
I tested by setting different permissions between locales, was working.

Reply:
A user can have only `edit-apostrophe-page` and still be granted commit capability, so I don't think this is right.

Test with `edit-apostrophe-page`, but with commit rights for the locale, and via an admin account specifically grant edit access to the page in question via its page settings (that's what edit-apostrophe-page requires before the user can actually edit a page).